src/nix/add-to-store.cc

```diff
@@ -21,6 +22,13 @@ struct CmdAddToStore : MixDryRun, StoreCommand
             .labels = {"name"},
             .handler = {&namePart},
         });
+
+        addFlag({
+            .longName = "flat",
+            .shortName = 0,
+            .description = "use flat file ingestion",
+            .handler = {&ingestionMethod, FileIngestionMethod::Flat},
+        });
```

Comment:
Maybe we should split `add-to-store` into `add-path` and `add-file`, analogous to `nix hash-file` and `nix hash-path`.

Reply:
Also we shouldn't take about "file ingestion" in the UI, since nobody will know what that means.